### PR TITLE
Design demo tweaks

### LINKED ISSUE #20: Build Accessibility page
Copy doc: https://docs.google.com/document/d/1_cCvuHSwS9i0pzD_4WHDFoTenVAfZVr9qGxG-rSHHGY/edit

### LINKED ISSUE #21: Build Coding standards page
Copy doc: https://docs.google.com/document/d/1lhD_rjaKk6OHk_yeXzEqFde5lqeZ7EmgmZl5BCfPwk8/edit

### LINKED ISSUE #23: Build Browser support page
Copy doc: https://docs.google.com/document/d/1flcSinNnjOJw88ooVQfUjQW8ZDHSXOe2cTHmgwIEx6I/edit#heading=h.k45g1u10i0ni

### LINKED ISSUE #32: Updates to current demo
There are still some issues with the demo:

http://vanillaframework.io-fix-design-review-points.demo.haus/

- [ ] The navigation is using a large font size than specified, it should be 14px https://github.com/ubuntudesign/vanilla-design/blob/master/Navigation/navigation.md
- [ ] The padding in the subscribe to newsletter input is still reversed - @pmahnke had changed this in Vanilla
- [ ] The link in the footer it should say "Vanilla Framework"
- [ ] Can we not have the #navigation-closed words in the URL after closing the nav?
- [ ] In small screen, there should be some space above the second hero button

Thanks.

### LINKED ISSUE #33: Include new pictograms in homepage hero
Links to assets: https://github.com/ubuntudesign/vanillaframework.io-design/issues/2#issuecomment-311885718

### LINKED ISSUE #34: Update layout of Get started row
Could we move the text in the Get started row, so it looks like this?

screen shot 2017-06-29 at 08 43 44

I placed the text in a col-5 and the code in a col-7 with prefix-1 - not sure if that's the most correct code. There should also be a link below the paragraph to the get started docs page - "Advanced instructions >"

@anthonydillon what could we say in there - there was no text before

### LINKED ISSUE #35: Add image to Guidelines row
Here is the asset: https://assets.ubuntu.com/v1/4a810e12-guidelines-illustration.svg

Can you place it in the available space, horizontally and vertically centred? The existing text block should stay as it is now.

Thanks.

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Capitalise Vanilla Framework link in footer
- Add correct pictograms
- Add Accessibility / Coding Standards / Browser support pages
- Make 'Getting started' content code block full width of page
- Add illustration to Guidelines row

## QA

- Pull code
- Run `./run serve --watch`
- Check site at http://0.0.0.0:8014/

Fixes #33, Fixes #34, Fixes #35, Fixes #32, Fixes #20, Fixes #21, Fixes #23

### CODE REVIEW

**Comment by barrymcgee** [comment]
@yaili Demo: http://barrymcgee.dreamhosters.com/vfio/

**Comment by yaili** [comment]
Some notes:
- [ ] Remove the Resources section from Accessibility guidelines
- [ ] Remove the Resources section from Browser support 
- [ ] When you go to one of the new pages, you should be able to go back to homepage by clicking on logo (doesn't work right now)
- [ ] On homepage Get started, change "Framrwork" to Framework
- [ ] Image in Guidelines row should be vertically centred
- [ ] Can we get the input padding fix in?
- [ ] The first and last items from https://github.com/canonical-websites/vanillaframework.io/issues/32 don't seem to be in this demo yet

Thank you

**Comment by barrymcgee** [comment]
Demo updated: http://barrymcgee.dreamhosters.com/vfio/

File: browser-support.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <li>Microsoft Edge</li>
+        <li>Apple Safari for iOS (latest)</li>
+        <li>Opera (latest stable)</li>
+        <li>Opera Mini (latest stable)</li>
+      </ul>
+      <p>We have a range of test devices that reflect the current marketplace and our own stats.</p>
+      <p>Last update: {{ page.sitemap.lastmod | date: '%B %d, %Y'}}</p>
+      <p>
+
+      </p>
```
Comment (anthonydillon) [inline browser-support.html]: Remove empty `<p>` tag.

File: accessibility.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      </ul>
+      <h3>Useful tools</h3>
+      <p>The web is abundant in tools that help to create and test for accessible sites. These are a few of the ones we consider the best and most useful ones:</p>
+      <ul>
+        <li><a href="https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb/related?hl=en">Chrome Accessibility Developer Tools</a></li>
+        <li><a href="http://wave.webaim.org/extension/">Web Accessibility Evaluation tool (WAVE) Chrome extension</a></li>
+        <li><a href="http://leaverou.github.io/contrast-ratio/">Contrast Ratio tool</a></li>
+        <li><a href="http://colororacle.org/">Color Oracle</a></li>
+        <li><a href="http://www.chromevox.com/">ChromeVox: a screen reader for Chrome</a></li>
+        <li><a href="https://tenon.io/index.php">Tenon.io: analyse your site’s accessibility</a></li>
```
Comment (anthonydillon) [inline accessibility.html]: Shouldn't these links have an external link on them?

**Comment by yaili** [comment]
- [ ] Navigation is now missing border bottom
- [ ] Navigation links not working
- [ ] Were we going to fix the Vanilla nav so you could click anywhere on the link area? It's not great usability that the link changes to underlined but you can't click it - how hard to fix?
- [ ] Reformat last updated dates to our copy style: 30 June 2017
- [ ] The pre tag in Get started in home should have no margin top
- [ ] u-equal-height in Guidelines in home should have no margin bottom as it affects the total padding of the strip
- [ ] Correct input field padding still not there
- [ ] footer top margin is affecting the perceived bottom padding of the Get involved/newsletter strip
- [ ] footer is not following spec: https://github.com/ubuntudesign/vanilla-design/tree/master/Footer - problem is particularly visible in top padding

**Comment by barrymcgee** [comment]
A lot of these issues are upstream issues so I've made the changes for the majority here while logging issues upstream.

One issue I've not fixed here but logged upstream is the clicking the nav item, not just the text. That issue is logged in https://github.com/vanilla-framework/vanilla-framework/issues/1154

**Comment by barrymcgee** [comment]
@yaili Input seems to be as the [spec](https://github.com/ubuntudesign/vanilla-design/blob/master/Forms/forms.md) says?

- Padding vertical: 8px
- Padding horizontal: 12px

screen shot 2017-07-05 at 15 30 50

**Comment by barrymcgee** [comment]
Updated demo: http://vanillaframework.io-32-design-demo-tweaks.demo.haus/

**Comment by yaili** [comment]
The input padding is the other way around, @pmahnke had submitted a fix for this.

**Comment by barrymcgee** [comment]
@yaili Input fix now in the demo.

**Pull request merged**